bestie/server/bigquery_metrics.go

```diff
@@ -0,0 +1,288 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	bq "cloud.google.com/go/bigquery"
+)
+
+type bigQuerySession struct {
+	ctx    context.Context
+	client *bq.Client
+}
+
+type bigQueryMetric struct {
+	metricname string
+	tags       string // stringified JSON
+	value      float64
+	timestamp  string // "YYYY-MM-DD hh:mm:ss.uuuuuu"
+}
+
+type bigQueryTable struct {
+	project   string
+	dataset   string
+	tablename string
+}
+
+type keyValuePair struct {
```

Comment:
These should probably look something like this

`type QueryTags map[string][]string `

Just a suggestion, but it makes it more readable and idiotmatic.

Reply:
Agreed. I've eliminated the extra type and incorporated `tags map[string]string` directly as an element of the `testMetric` struct.